scribejava-apis/src/main/java/com/github/scribejava/apis/polar/PolarOAuth20Service.java

```diff
@@ -0,0 +1,45 @@
+package com.github.scribejava.apis.polar;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.httpclient.HttpClient;
+import com.github.scribejava.core.httpclient.HttpClientConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.model.OAuthRequest;
+import com.github.scribejava.core.oauth.AccessTokenRequestParams;
+import com.github.scribejava.core.oauth.OAuth20Service;
+import com.github.scribejava.core.pkce.PKCE;
+
+import java.io.OutputStream;
+
+public class PolarOAuth20Service extends OAuth20Service {
+
+	public PolarOAuth20Service(DefaultApi20 api, String apiKey, String apiSecret, String callback, String defaultScope, String responseType,
+		OutputStream debugStream, String userAgent, HttpClientConfig httpClientConfig, HttpClient httpClient) {
+		super(api, apiKey, apiSecret, callback, defaultScope, responseType, debugStream, userAgent, httpClientConfig, httpClient);
+	}
+
+	@Override
+	protected OAuthRequest createAccessTokenRequest(AccessTokenRequestParams params) {
```

Comment:
This override is unneeded. It's the only diff is using "request.addBodyParameter" instead of "request.addParameter". But you didn't override getAccessTokenVerb in API. It's POST by default. So "request.addParameter" will invoke "request.addBodyParameter" inside.

ps.The second diff is questionable. You skipped setting OAuthConstants.SCOPE parameter. why?

Reply:
Yeah, the `request.addBodyParamete` was indeed not needed for POST requests. I've reverted that. However the `OAuthConstants.SCOPE` has to be left out for access token request otherwise the Polar API will respond with a 400, as described in their API https://www.polar.com/accesslink-api/#token-endpoint